src/core/client/admin/components/MediaContainer/YouTubeMedia.tsx

```diff
@@ -6,7 +6,7 @@ import { BaseButton, Flex, Icon } from "coral-ui/components/v2";
 
 import styles from "./Media.css";
 
-interface Props {
+export interface Props {
```

Comment:
Typically if you're exporting `Props`, I'd name it after the component to give it a unique name (i.e. `YouTubeMediaProps` instead).